### PR TITLE
Minor updates to "Testing on a Quest"

### PR DESCRIPTION
**Pull request by TheCzar1994**
SideQuest apparently has an "Experimental Web Installer" now. This clarifies to use the "Advanced Installer" compatible with these instructions. Also removes "1. Locate your WIP song folder on your PC." as this is being explained following it so it seems redundant and then re-numbers the rest.

### CODE REVIEW

File: wiki/mapping/index.md [hunk 1]
```diff
@@ -390,15 +390,19 @@ Your changes will be loaded without needing to restart the game!
 Follow these steps to test any of your maps made with a community editor using an Oculus Quest. You must have
 [SideQuest](https://sidequestvr.com) and [BMBF](https://bmbf.dev/stable) installed.
 
-1. Locate your WIP song folder on your PC.
-2. Establish a connection to your Quest from your PC with a USB cable.
-3. Open SideQuest on your PC and click the folder icon on the top right.
+:::tip NOTE
+When [downloading SideQuest](https://sidequestvr.com/setup-howto), be sure to download the "Advanced Installer". We'll
+walk you through how to use it below.
```
Comment (bloodcloak) [inline wiki/mapping/index.md:395]: ```When [downloading SideQuest](https://sidequestvr.com/setup-howto), be sure to download the `Advanced Installer`.```

File: wiki/mapping/index.md [hunk 2]
```diff
@@ -390,15 +390,19 @@ Your changes will be loaded without needing to restart the game!
 Follow these steps to test any of your maps made with a community editor using an Oculus Quest. You must have
 [SideQuest](https://sidequestvr.com) and [BMBF](https://bmbf.dev/stable) installed.
 
-1. Locate your WIP song folder on your PC.
-2. Establish a connection to your Quest from your PC with a USB cable.
-3. Open SideQuest on your PC and click the folder icon on the top right.
+:::tip NOTE
```
Comment (bloodcloak) [inline wiki/mapping/index.md:393]: `:::warning NOTE`

**Review by bloodcloak — CHANGES_REQUESTED** [review]

**Comment by bloodcloak** [comment]
Requesting since I'm not able to push the format suggestion to the branch

**Comment by TheCzar1994** [comment]
Sorry about that! Should be able to in the future if needed but I made the suggested changes.

**Review by bloodcloak — APPROVED** [review]

**Pull request merged**